src/Workspaces/Core/Portable/Workspace/Host/PersistentStorage/IChecksummedPersistentStorage.cs

```diff
@@ -40,24 +40,24 @@ internal interface IChecksummedPersistentStorage : IPersistentStorage
         /// is provided, the persisted checksum must match it.  If there is no such stream with that name, or the
         /// checksums do not match, then <see langword="null"/> will be returned.
         /// </summary>
-        Task<Stream> ReadStreamAsync(string name, Checksum checksum = null, CancellationToken cancellationToken = default);
+        Task<Stream> ReadStreamAsync(string name, Checksum? checksum = null, CancellationToken cancellationToken = default);
```

Comment:
>null [](http://example.com/codeflow?start=71&length=4)

Is Checksum.Null not usable here instead of making the param type nullable?

Reply:
Yeah. Let me try that!